beebop/app.py

```diff
@@ -229,6 +292,13 @@ def get_results(type):
 
 
 def get_clusters_internal(hash, redis):
+    """
+    returns cluster assignment results
+
+    Arguments:
+    hash - project hash
```

Comment:
Project hash should go in the glossary too so that we can keep the short description here